packages/react/spec/components/SignedInOrRedirect.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import "@testing-library/jest-dom";
+import { render } from "@testing-library/react";
+import React from "react";
+import { superAuthApi } from "../../spec/apis";
+import { TestWrapper } from "../../spec/testWrapper";
+import { expectMockSignedInUser, expectMockSignedOutUser } from "../../spec/utils";
+import { SignedInOrRedirect } from "../../src/components/SignedInOrRedirect";
+
+describe("SignedInOrRedirectOrRedirect", () => {
+  const { location } = window;
+  const mockAssign = jest.fn();
+
+  beforeAll(() => {
+    // @ts-expect-error mock
+    delete window.location;
+    // @ts-expect-error mock
+    window.location = { assign: mockAssign };
```

Comment:
`jest.spyOn(window.location, "assign")` no worky? I don't super care just curious

Reply:
I encountered this problem: https://remarkablemark.org/blog/2018/11/17/mock-window-location/#spy-on-method

This is the workaround for jsdom 14+